src/components/domain/Others/index.jsx

```diff
@@ -0,0 +1,31 @@
+import { ContentContainer } from '@components/base';
+import { ContentItem } from '@components/base';
+import { Toggle } from '@components/base';
+import { BORDER_STYLE } from '@utils/constants';
+import styled from 'styled-components';
+
+const Others = () => {
+  return (
+    <Wrap>
+      <ContentContainer title="기타 설정" />
+      <ContentItem title="감사카드 제공">
+        <StyledToggle>
+          <Toggle/>
+        </StyledToggle>
+      </ContentItem>
+    </Wrap>
```

Comment:
이것도 위랑 마찬가지로 사용해주시면 돼요!
```suggestion
    <ContentContainer title="기타 설정">
      <ContentItem title="감사카드 제공">
        <StyledToggle>
          <Toggle/>
        </StyledToggle>
      </ContentItem>
    </ContentContainer>
```